org.civicrm.hremergency/CRM/Hremergency/Upgrader/Steps/1000.php

```diff
@@ -12,30 +12,42 @@ public function upgrade_1000() {
     // relationship with employee options to create
     $relationships = ['Friend', 'Spouse', 'Partner'];
 
-    // fetch 'Relationship with employee' option group name
-    $groupName= civicrm_api3('CustomField', 'get', [
-      'sequential' => 1,
-      'return' => ['column_name'],
-      'name' => 'Relationship_with_Employee',
-      'options' => ['limit' => 1]
-    ]);
-
     // create required options
-    if (!empty($groupName['values']['column_name'])) {
-      $groupName = $groupName['values']['column_name'];
-
-      foreach($relationships as $relationship) {
+    foreach($relationships as $relationship) {
+      if (!$this->up1000_valueExists($relationship, 'relationship_with_employee_20150304120408')) {
         civicrm_api3('OptionValue', 'create', [
           'sequential' => 1,
-          'option_group_id' => $groupName,
+          'option_group_id' => 'relationship_with_employee_20150304120408',
           'name' => $relationship,
           'label' => $relationship,
-          'is_active' => 1,
+          'value' => strtolower($relationship),
+          'is_active' => 1
         ]);
       }
     }
 
     return TRUE;
   }
 
+  private function up1000_valueExists($value, $groupName) {
```

Comment:
up1000_valueExists doesn't mean anything , a better name would probably be : 

up1000_IsOptionValueExist